plugin-dev/Source/Sentry/Private/Desktop/SentrySubsystemDesktop.cpp

```diff
@@ -154,6 +154,22 @@ void SentrySubsystemDesktop::InitWithSettings(const USentrySettings* settings, U
 #endif
 	}
 
+	switch (settings->DatabaseLocation)
+	{
+	case ESentryDatabaseLocation::ProjectDirectory:
+		databaseParentPath = FPaths::ProjectDir();
+		break;
+	case ESentryDatabaseLocation::ProjectUserDirectory:
+		databaseParentPath = FPaths::ProjectUserDir();
+		break;
+	}
+
+	if(databaseParentPath.IsEmpty())
+	{
+		UE_LOG(LogSentrySdk, Warning, TEXT("Unknown Sentry database location. Falling back to FPaths::ProjectUserDir()."));
+		databaseParentPath = FPaths::ProjectUserDir();
+	}
```

Comment:
AFAIK, other SDKs would interpret a missing or invalid database path as having the feature disabled. But there it's most likely about event-offline-caching.
I guess sentry-native would not do much without it?

Reply:
`sentry-native` database path will default to the current working directory if one is not provided explicitly so technically it should be able to continue working as usual. On the other hand, if it's invalid rather than missing native SDK initialization will fail. Anyway, having the above check in place should prevent us from getting into any of these situations.